.env.railway

```
@@ -0,0 +1,38 @@
+# Configuración del servidor SMTP para envío de emails
+SMTP_HOST=smtp.gmail.com
+SMTP_PORT=587
+SMTP_USER=eltallerdelsoftware@gmail.com
+SMTP_PASS="wrtp qkqn sqic qyiu"
+SMTP_FROM_EMAIL=eltallerdelsoftware@gmail.com
+SMTP_FROM_NAME="Gestor de Contraseñas - Astillero La Roca"
+
+# Configuración de sesión
+SESSION_DURATION=1800 # 30 minutos de inactividad (en segundos)
+SESSION_COOKIE_NAME=ASTILLERO_SESSION
+SESSION_COOKIE_SECURE=false # Cambiar a true en producción con HTTPS
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SESSION_COOKIE_SECURE key should go before the SESSION_DURATION key

(UnorderedKey)